bin/utils.sh

```diff
@@ -40,6 +31,26 @@ download_url() {
     version="$2"
   fi
 
+  case "$(uname -s)" in
+    "Linux")
+      platform=linux64
+      ;;
+    "Darwin")
+      if [ "$version" = "nightly" ]; then
```

Comment:
Unfortunately, `neovim` has not retroactively published binaries for the different architectures on previous stable releases.